daemon/daemon_unix.go

```diff
@@ -550,7 +550,7 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	// check for various conflicting options with user namespaces
 	if daemon.configStore.RemappedRoot != "" && hostConfig.UsernsMode.IsPrivate() {
 		if hostConfig.Privileged {
-			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces")
+			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces.  To temporarily disable user namespaces, use '--userns=host'.")
```

Comment:
> can we open a discussion on how to design that ?

Yes, we can open a discussion on that; however, given that the plan is to move to gRPC API's, we should take care not to do a massive overhaul of the API for this; perhaps it can be handled as part of the migration.

> How about if it said you must run the usernamespace in the host namespace?

Did you mean to say `you must run the container in the host namespace`? That sounds good to me

Looking at the current changes; we should not include "temporarily" in the error description, because it's not temporarily for the container, it's just that you start a specific container in the host namespace instead of user namespace.

Does this help @TomSweeneyRedHat ? Otherwise happy to try to come up with additional suggestions 👍